learnr_bank/learnr_set2.Rmd

```diff
@@ -0,0 +1,468 @@
+```
+
+### Set 1: Answer Key
+
+``` {r, func_q1-answer}
+#Solution to Question 1
+sqrt(206)
+```
+
+``` {r, func_q2-answer}
+#Solution to Question 2
+log(206)
+```
+
+``` {r, func_q3-answer}
+#Solution to Question 3
+log(206, 10)
+```
+
+``` {r, func_q4-answer}
+#Solution to Question 4
+ class(7)
+```
+
+``` {r, func_q5-answer}
+#Solution to Question 5
+class("7")
+```
+
+``` {r, func_q6-answer}
+#Solution to Question 6
+x <- 7
+type_x <- class(x)
+type_x == "numeric"
+```
+
+### Set 2: Working with arrays in R
+
+Please name these questions `array_qX` where X is the number (1, 2, 3, ...).
+
+> Note: The terms "array" and "vector" both refer to the same thing and are used interchangeably, but a "list" is something else entirely! These exercises don't consider lists at all.
+
+
+1. Define an array variable called `array_numbers` that contains the values 1, 2, 3, 4, and 5. Then, type the variable name itself to make sure its contents look right.
+  + **Hint:** Use the function `c()` ("combine") to define your array
+  + Solution: array_numbers <- c(1,2,3,4,5)
+              array_numbers
+ 
+``` {r array_q1, exercise = TRUE}
+
+```          
+
+<div id= "array_q1-hint">
+**Hint:** Use the function `c()` ("combine") to define your array
+</div>
+
+2. Define an array variable called `array_numbers2` that again contains the values 1, 2, 3, 4, and 5, **but!** this time use a colon `:` instead of `c()`. Again, type the variable name itself to make sure its contents look right.
+  + **Hint:** `5:7` defines an array with values 5, 6, 7. Can you adapt this approach to the given question
+  + Solution: array_numbers2 <- 1:5
+              array_numbers2
+              
+``` {r array_q2, exercise = TRUE}
+
+```  
+
+<div id= "array_q2-hint">
+**Hint:**  `5:7` defines an array with values 5, 6, 7. Can you adapt this approach to the given question
+</div>
+
+3. Define an array variable called `even_numbers` that contains the values 2, 4, 6, 8, and 10 using the `c()` function.
+  + (no hint)
+  + Solution: even_numbers <- c(2,4,6,8,10)
+              even_numbers
+
+``` {r array_q3, exercise = TRUE}
+
+```  
+
+4. Define an array variable called `even_numbers` that contains the values 2, 4, 6, 8, and 10 using a new function (!) `seq()`. The chunk below shows an example of using `seq()`. Run this code to see the function in action, and then *adapt* the code to make the `even_numbers` array.
+
+  + **Hint 1**: You may want to run `?seq` to see more about how the function works!
+  + **Hint 2**: Always print out your output to be sure the variable you've created is correct. Keep trying until you get there - coding is all about trial and error! 
+  + Solution: even_numbers <- seq(2, 10, 2)
+              even_numbers
+
+```{r array_q4, exercise = TRUE}
+seq(1, 34, by = 3)
```

Comment:
Was this one of the questions where you wanted to have a baseline code? You did not specifically tell me to add it, but the question seemed to imply it?

Reply:
Yes, I did mean to include baseline - wanna make something up for it? :)